sdk/identity/azure-identity/src/main/java/com/azure/identity/ManagedIdentityCredential.java

```diff
@@ -211,6 +212,8 @@ public Mono<AccessToken> getToken(TokenRequestContext request) {
                             ? "DefaultAzureCredentialBuilder."
                             : "ManagedIdentityCredentialBuilder."))));
             }
+
+            LOGGER.info("User-assigned Managed Identity ID: " + getClientId());
```

Comment:
should this be VERBOSE ?

Reply:
@christothes has it at informational so that's where I put it, but I don't have a strong opinion. What do you think Chris?

Reply:
The default verbosity for our listeners is Info, so I like that better. Verbose is a bit obscure, and I don't think this is overly noisy.